netket/operator/_kinetic.py

```diff
@@ -20,10 +20,35 @@
 import jax.numpy as jnp
 
 from netket.utils.types import DType, PyTree, Array
+import netket.jax as nkjax
 from netket.hilbert import AbstractHilbert
 from netket.operator import ContinuousOperator
 
 
+def jacrev(f):
```

Comment:
can you remove those changes and move them to another PR?

Reply:
ok